src/digodoc_lib/module.ml

```diff
@@ -20,13 +20,15 @@ let long_name ~mdl_name ~mdl_opam =
   mdl_opam.opam_name ^ "::" ^ mdl_name
 
 let file mdl ~ext =
-  mdl.mdl_dir.dir_name // ( mdl.mdl_basename ^ ext )
+  let name_n_ext = mdl.mdl_basename ^ ext in
+  let path = StringMap.find ext mdl.mdl_path in
+  path // name_n_ext
```

Comment:
Don't need to concat this. Values inside ```mdl.mdl_path``` are the paths to the module but not to the directory containig module with given extension.